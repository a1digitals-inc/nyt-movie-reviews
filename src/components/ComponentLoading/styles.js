import styled from 'styled-components';

export const Container = styled.div`
  margin-top: 5px;

  img {
    margin: 0 !important;
    height: 20px !important;
  }
`;
