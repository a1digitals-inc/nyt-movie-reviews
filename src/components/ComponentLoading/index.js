import React from 'react';

import { Container } from './styles';
import Loading from '../Loading';

export default function PageLoading() {
  return (
    <Container>
      <Loading />
    </Container>
  );
}
