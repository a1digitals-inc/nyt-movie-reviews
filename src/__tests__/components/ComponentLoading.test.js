import React from 'react';
import { shallow } from 'enzyme';

import ComponentLoading from '../../components/ComponentLoading';

describe('ComponentLoading component', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<ComponentLoading />);
    expect(wrapper).toMatchSnapshot();
  });
});
