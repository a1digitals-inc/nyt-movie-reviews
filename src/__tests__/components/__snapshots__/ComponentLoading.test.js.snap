// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentLoading component should render without crashing 1`] = `
<styled.div>
  <Loading />
</styled.div>
`;
